kivy/core/window/__init__.py

```diff
@@ -115,7 +115,7 @@ class Keyboard(EventDispatcher):
         'é': 233, 'è': 232,
     }
 
-    __events__ = ('on_key_down', 'on_key_up', 'on_textinput')
+    __events__ = ('on_key_down', 'on_key_up', 'on_textinput','on_textedit')
```

Comment:
please add space after `,`.
```suggestion
    __events__ = ('on_key_down', 'on_key_up', 'on_textinput', 'on_textedit')
```